parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -253,6 +251,11 @@ def __init__(
             }
         )
 
+    def process_left_pane_text(self, args: "DictConfig") -> str:
```

Comment:
hmm, honestly, I think that these 3 lines of code will probably be the same for all subclasses of model_chat, meaning that the lines will need to get repeated in all subclasses. I think it'll lead to less duplicated code if, instead, there's a method called something like `format_left_pane_text()`, which you just define as `pass` for this superclass

Reply:
Resolved